packages/amplify-graphql-transformer-core/src/transform-host.ts

```diff
@@ -131,6 +131,7 @@ export class DefaultTransformHost implements TransformHostProvider {
   public addResolver(
     typeName: string,
     fieldName: string,
+    resolverLogicalId: string,
```

Comment:
Should this be an option param or is it required in all cases?

Reply:
to me it looks that is should be required in all cases

Reply:
I agree with @edwardfoyle we should consider it option since there a good chunk of instances where we use the actual typeName and fieldName, in areas where don't actually provide the typeName we should use the resolverLogicalId as an override.

Reply:
updated